packages/sing-box/build.sh

```diff
@@ -2,10 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://sing-box.sagernet.org
 TERMUX_PKG_DESCRIPTION="The universal proxy platform"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="kay9925@outlook.com"
-TERMUX_PKG_VERSION="1.2.3"
+TERMUX_PKG_VERSION="1.3-beta1"
 TERMUX_PKG_SRCURL="https://github.com/SagerNet/sing-box/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=a12b81950deb181cf2c1783685e0dd66502376471db57e3787fad3f3e27fd48c
+TERMUX_PKG_SHA256=0c3652b1aab3659d5f45d9f3d3090c3385e0a11ef345942c71ba883d43bab9e2
 TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
Auto-update would not update to 1.3-beta1 because it is not the "latest release".

Reply:
So it looks like I can't enable TERMUX_PKG_AUTO_UPDATE=true, I'll subscribe to sing-box release updates and update this repository manually

Reply:
> So it looks like I can't enable TERMUX_PKG_AUTO_UPDATE=true, I'll subscribe to sing-box release updates and update this repository manually
Usually, versions marked with `beta` or `dev` are not official releases
auto update will update to the version set to `latest` .
![image](https://user-images.githubusercontent.com/57583560/230701265-8dfa9e0f-dab1-4e74-9f8a-e5cef0ebe721.png)

Reply:
For sing-box, a beta tag may be a stable version

Reply:
> For sing-box, a beta tag may be a stable version

All beta versions will become official versions after a period of time.
`beta` or `dev` releases are usually unstable or has bug.

Reply:
Well, this pr can no longer be merged and I will enable TERMUX_PKG_AUTO_UPDATE in the next release when I submit the pr.